### PR TITLE
Individual Rake Tasks

### PR DESCRIPTION
**Pull request by slotaj**
@bethsecor 

@rrgayhart 

We now have our rake tasks configured so that each task can be ran separately on local machines for testing. Additionally the all_jobs still exists for our daily scheduler.

### CODE REVIEW

File: lib/tasks/job_fetch.rake [hunk 1]
```diff
@@ -1,5 +1,5 @@
 namespace :job_fetch do
-  desc "Run the stack overflow fetcher"
+  desc "Run all job fetchers"
 
   task all_jobs: :environment do
```
Comment (rrgayhart) [inline lib/tasks/job_fetch.rake]: To keep this DRY - this should probably call the other tasks directly:

Check out some ways to do that here: http://stackoverflow.com/questions/577944/how-to-run-rake-tasks-from-within-rake-tasks

**Comment by slotaj** [comment]
@rrgayhart I implemented a solution that seems to be working, I've tested it and weworkremotely is bringing back jobs, when I run both the services. Also dry solution has been implemented.

File: lib/tasks/job_fetch.rake [hunk 2]
```diff
@@ -1,11 +1,27 @@
...
+
+  task stackoverflow: :environment do
     Technology.find_each do |t|
       StackOverflow.new(t.name).scrape
     end
+  end
+
+  desc "we work remotely job fetcher"
+
+  task weworkremotely: :environment do
```
Comment (bethsecor) [inline lib/tasks/job_fetch.rake]: Can you still run the individual rake tasks with this implementation?

**Comment by rrgayhart** [comment]
Looks great - provided Beth's question is resolved.

Now you just need to squash down the commits

**Comment by slotaj** [comment]
@bethsecor @rrgayhart Merged master and there are no merge conflicts, branch should be ready to merge. Attempted to squash commits with merge master, and it had issues with finding most recent commits. On advice of Beth, pushing with two commits because merged master may have caused issues.

**Comment by rrgayhart** [comment]
:+1: :ship: Awesome!

**Pull request merged**